.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+<!-- First of all, check the troubleshooting wiki page for common issues at:
+https://github.com/kulshekhar/ts-jest/wiki/Troubleshooting -->
+
+## Issue <!-- describe the issue below -->:
+
+
+## Expected behavior <!-- describe the expected behavior below -->:
+
+
+## Output from debug log <!-- You can activate the debug logger by setting the environment variable TS_JEST_DEBUG="true" before running yarn test. The output of the logger will be in **<your_project_dir>/node_modules/ts-jest/debug.txt**, paste it below -->:
+```bash
+# content of debug.txt :
+
+```
+
+
+## Minimal repo <!-- If you haven't already, create the smallest possible repo that reproduces this issue by running `npm install` and `npm test`. This will speed up any fixes that this issue might need. Paste the minimal repo URL below -->:
+
+
+<!-- Optional (but highly recommended): Configure Travis (or your favorite system) with the minimal repo. This allows potential solutions to be tested against the minimal repo. This saves everyone time and avoids a lot of back and forth. -->
```

Comment:
Here I've added the first comment. Also while keeping original texts I've put between comments the stuff that people use to leave in the issue text. Also changed from a bullet-list to bigger headers so that the issue is more readable.